mopidy_tidal/library.py

```diff
@@ -18,6 +18,24 @@
 
 logger = logging.getLogger(__name__)
 
+class Cache(dict): 
+    '''cache class, requires python3 style (ordered) dicts'''
```

Comment:
If this was a an actual `OrderedDict` you could use [`popitem(last=False)`](https://docs.python.org/3.7/library/collections.html#collections.OrderedDict.popitem) to simplify your `__prune()`.

Reply:
I'm relying on Python 3 dicts (which are essentially `OrderedDict`).  For some reason, I thought Mopidy was Python3 only?

I'll get this changed in any case.

Reply:
Updated.

Reply:
You are totally correct that Mopidy requires Python 3.7+ and in Python 3.7, dict became ordered. But dict doesn't have all the same functionality as `OrderedDict`. Popping from the front with `last=False` is only possible with  `OrderedDict`.

Reply:
I saw that, which is why I changed it.